External/Plugins/HaXeContext/Context.cs

```diff
@@ -577,6 +580,21 @@ public override void RemoveClassCompilerCache()
         }
         #endregion
 
+        #region SDK
+        private InstalledSDK GetCurrentSDK()
+        {
+            return hxsettings.InstalledSDKs.FirstOrDefault(sdk => sdk.Path == currentSDK);
```

Comment:
I quite like it. I'm a big fan of IntelliJ and it allows you to easily configure a lot of shortcuts the same way. The code analysis is really helpful too (convert to LINQ expressions like here, warns about potential null errors, unused usings / variables / functions etc).

Some inspections can be a bit annoying (the naming convention rules especially, since FD doesn't match most of those, but they can be turned off or adjusted).